agent/src/capability/capability.cpp

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Yes. I mean you have two simple options: have the capabilities merge themselves (if they are captured in classes), or have some method in the config store that does the merging.